apps/challenges/migrations/0062_challenge_slack_webhook_url.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please rename this file to `0062_add_slack_webhook_url_field_in_challenge_model.py`.